CHANGELOG.md

```diff
@@ -4,6 +4,9 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog].
 
 ## Unreleased
+### Enhancements
+* Support multiple formatters ([#31]).
```

Comment:
Can we include a brief explanation of this feature? (You can now use a list in `apheleia-mode-alist`, and all the formatters listed will be run in sequence.)